static/includes/TrueCommand/2.0/TrueCommandLDAP.md.part

```diff
@@ -15,6 +15,54 @@ Multiple LDAP servers and Domains can be added.
+| User ID Field (string) | Domain fieldname to use for user-matching. The default value is *User ID FIELD*. Another field commonly-used is "cn" (common name) |
+| Group ID Field (string) | The domain fieldname to use when searching for a group name. The default value is "cn" (common name). |
+| BIND User Domain (string) | <span>The full domain setting for a pre-authenticated bind to the server.<br>Example: "uid=binduser,cn=read-only-bind,dc=mycorp,dc=com"<br>For an unauthenticated bind set this field to just a name (example: "truecommand-bind"). This is sometimes used for logging purposes on the LDAP, but otherwise is not validated.</span> |
+| Bind Password (string) | The password to use for the bind user For an unauthenticated bind, leave this field blank while setting the *BIND User Domain* to a non-empty value. |
+
+### LDAP connection options
+
+There are two common methods of validating an LDAP user which TrueCommand supports:
+
+*Direct Bind*
+In this method, the *Domain* and *User ID Field* are used to create a static domain string which is then utilized to attempt to authenticate the user.
+
+Example:
+
+* Domain: "dc=mycorp,dc=com"
+* User ID Field: "uid" 
+
+When user "bobby.singer" attempts to login, TrueCommand will establish an SSL-secure connection to the LDAP server and then attempt to bind with the static domain "uid=bobby.singer,dc=mycorp,dc=com" and the user-provided password. If successful, then the user authentication has been verified and Bobby Singer is allowed access to TrueCommand.
+
+*Indirect Bind*
+This method of authentication is much more dynamic and searches for the proper user domain settings rather than making assumptions about the format. 
+In this situation, a "bind user" (typically a read-only, minimal-permissions user account) is configured with a known domain/password to perform the initial bind to the LDAP server. 
+Once logged in, a search is then performed to find the domain of the user currently requesting to login, and then a second bind is attempted utilizing the user domain and provided password.
+
+Example:
+
+* Domain: "dc=mycorp,dc=com"
+* User ID Field: "uid"
+* BIND User Domain: "uid=binduser,cn=read-only-bind,dc=mycorp,dc=com"
+* BIND Password: "pre-shared-key"
+
+When "bobby.singer" attempts to login, TrueCommand will establish an SSL-secure connection to the LDAP server, and then the *BIND User Domain* and *BIND Password* settings are used to perform an initial bind utilizing pre-known settings from your LDAP provider. Once bound, a search is then performed for the user matching "uid=bobby-singer", but only within the subdomains that include the "domain" setting ("dc=mycorp,dc=com" in this example). If a user is found, the full user domain string from the search result is then used to initialize a second bind along with the user-provided password. If successful, then the user authentication has been verified and Bobby Singer is allowed access to TrueCommand.
+
+
+### SSL/TLS Connection Info
+
```

Comment:
Put a little warning box here stating that SSL connections are *required* for AD/LDAP authentication.

Reply:
I have added the warning